index.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable no-unused-vars */
+import { Book } from './modules/book-class.js';
+import { clearFields } from './modules/clear-fields.js';
+import { DateTime } from './modules/luxon.min.js';
+
+if (localStorage.getItem('My Books') === null) {
+  localStorage.setItem('My Books', JSON.stringify([]));
+}
+
+const Localstoragebook = JSON.parse(localStorage.getItem('My Books'));
+
+const updateLocalStorage = () => {
+  localStorage.setItem('My Books', JSON.stringify(Localstoragebook));
+};
+
+const createListOfBooks = (arr) => {
+  let books = '';
+  for (let i = 0; i < arr.length; i += 1) {
+    let liClass = 'dark-bakcground';
+    if (i % 2 === 0) {
+      liClass = 'book-li';
+    }
+    books += `
+                <li class= '${liClass}'>${arr[i].title} by ${arr[i].author} <button class="remove-btn" onclick="removeBook(${i})">Remove</button></li> <br />
+                `;
+  }
+  return books;
+};
+
+const showBooks = () => {
+  const listOfBooks = document.querySelector('.container');
+  listOfBooks.innerHTML = `
+                <ul class="book-ul"/>
+                ${createListOfBooks(Localstoragebook)}</ul>
+            `;
+};
+
+const addNewBook = (bookTitle, bookAuthor) => {
+  const myBook = new Book(bookTitle, bookAuthor);
+  Localstoragebook.push(myBook);
+  updateLocalStorage();
+  showBooks();
+  clearFields();
+};
+
+const removeBook = (i) => {
+  Localstoragebook.splice(i, 1);
+  updateLocalStorage();
+  showBooks();
+  clearFields();
+};
```

Comment:
- [ ] to make your ` index.js` clean as the entry file for your project I would highly suggest that you refactor your code and move your **_Book management functions_**  to a custom module example: `book_management.js` which is gonna be responsible for all Books management logic  that way you will have more reasonable modules and clean/organized code .

**_Note: implementing them in your book class is a great practice as well_**